src/actions/appSettingsActions.js

```diff
@@ -179,3 +180,24 @@ export const setUserJoinedBetaAction = (userJoinedBeta: boolean, ignoreSuccessTo
     });
   };
 };
+
+
+export const changeAppThemeAction = () => {
+  return (dispatch: Dispatch, getState: GetState) => {
+    const {
+      appSettings: { data: { theme: { mode } } },
```

Comment:
if changing constants to `_THEME` then maybe this can be
```suggestion
      appSettings: { data: { theme: { current } } },
```